.github/workflows/codeql.yml

```diff
@@ -18,7 +22,7 @@ on:
     # The branches below must be a subset of the branches above
     branches: [ main ]
   schedule:
-    - cron: '40 8 * * 5'
+    - cron: 40 8 * * 5
```

Comment:
I know this was already here, but this cron job time is pretty random. 8:40am on Fridays. The time is arbitrary, but I'm open for however frequent/infrequent you want to have it run